core/packages/test/scripts/configure-contracts.sh

```diff
@@ -25,7 +25,7 @@ configure_contracts()
     	"initialize(uint64,(uint128,uint128,bytes32),(uint128,uint128,bytes32))" \
     	--rpc-url $eth_endpoint_http \
     	--private-key $PRIVATE_KEY \
-      $beefy_start_block \($current_id,$current_length,$current_root\) \($next_id,$next_length,$next_root\)
+      $beefy_start_block \($current_id,$current_length,$current_root\) \($next_id,$next_length,$next_root\) || true
```

Comment:
```suggestion
      $beefy_start_block \($current_id,$current_length,$current_root\) \($next_id,$next_length,$next_root\) || true
      # "|| true" can be removed once https://github.com/foundry-rs/foundry/pull/4010 has been released
```

Reply:
Good catch! IMO though foundry is fast but not stable enough, weird issue sometimes. So I'll keep it temprarily and test in a seperate PR.